recipes/blast/meta.yaml

```diff
@@ -10,43 +10,57 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - boost
     - zlib
     - bzip2
-    - gnutls
-    - gmp
-    - nettle 3.4
     - pcre
+    - entrez-direct
+    - curl
```

Comment:
So blast needs curl now? Does it link against it? If so, might need to pin.

Reply:
`update_blastdb.pl` uses it.

Reply:
Ah, so cmdline. No pinning then.